templates/components/page/contact-us-form.html

```diff
@@ -50,7 +50,7 @@
     {{#if forms.contact.captcha_url}}
         <div class="form-field">
             <label class="form-label" for="captcha">{{lang 'forms.contact_us.captcha'}}</label>
-            <img src="{{forms.contact.captcha_url}}" alt="img">
+            <img src="{{forms.contact.captcha_url}}" alt="img" title="img">
```

Comment:
Same as above, `alt` and `img` should be meaningful. So it should probably be something about "captcha" instead of "img".